<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(223, 221, 220);
color:rgb(255, 255, 255)</string>
  </property>
  <widget class="QPushButton" name="labAddTeacher">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(223, 221, 220);
color:rgb(0, 0, 0);

font: 18pt &quot;Mohammad Annoktah&quot;;</string>
   </property>
   <property name="text">
    <string>إضافة معلم جديد</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTeacherMajor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTeacherFName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الاسم </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTeacherPhone">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الهـــــاتف</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الـتـخــصـص</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveTeacher">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>690</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 170, 127);
color:rgb(255, 255, 255);
font: 15pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="text">
    <string>حفـــــظ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>المهمـــــــة</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAddingTeacher">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>690</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(194, 64, 0);
color:rgb(255, 255, 255);
font: 15pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="text">
    <string>إلغــــــاء</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>تاريخ الميلاد</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateTeacherDOB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTeacherLName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>اللقب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الوردية</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combShiftsType">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>الاولى</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>الثانية</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>الاولى والثانية</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>تاريخ التخرج</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>المؤهـــــــل</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTeacherQualification">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateTeacherDOQualification">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>550</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>سنوات الخبرة</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtExceperianceYears">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
   </property>
   <property name="placeholderText">
    <string>قم بأدخال قيمة رقمية</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboTeacherTask">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>مدير</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>مدرس</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>وكيل</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>مدير النظام</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الحالة</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboTeacherStatus">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>600</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>مفعل</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>موقف</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtTeacherFName</tabstop>
  <tabstop>txtTeacherLName</tabstop>
  <tabstop>txtTeacherPhone</tabstop>
  <tabstop>dateTeacherDOB</tabstop>
  <tabstop>txtTeacherMajor</tabstop>
  <tabstop>labAddTeacher</tabstop>
  <tabstop>btnSaveTeacher</tabstop>
  <tabstop>btnCancelAddingTeacher</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
